drivers/gpio/gpio_mcux_lpc.c

```diff
@@ -44,27 +43,34 @@ static int gpio_mcux_lpc_configure(struct device *dev,
 				   int access_op, u32_t pin, int flags)
 {
 	const struct gpio_mcux_lpc_config *config = dev->config->config_info;
+	struct gpio_mcux_lpc_data *data = dev->driver_data;
 	GPIO_Type *gpio_base = config->gpio_base;
+	u32_t port = config->port_no;
 
-	/* Check for an invalid pin configuration */
-	if ((flags & GPIO_INT) && (flags & GPIO_DIR_OUT)) {
-		return -EINVAL;
+	if (((flags & GPIO_INPUT) != 0) && ((flags & GPIO_OUTPUT) != 0)) {
+		return -ENOTSUP;
 	}
 
-	/* Check if GPIO port supports interrupts */
-	if (flags & GPIO_INT) {
+	if ((flags & GPIO_SINGLE_ENDED) != 0) {
 		return -ENOTSUP;
 	}
 
 	/* supports access by pin now,you can add access by port when needed */
 	if (access_op == GPIO_ACCESS_BY_PIN) {
-		/* input-0,output-1 */
-		if ((flags & GPIO_DIR_MASK) == GPIO_DIR_IN) {
-			gpio_base->DIR[config->port_no] &= ~(BIT(pin));
-		} else {
-			gpio_base->SET[config->port_no] = BIT(pin);
-			gpio_base->DIR[config->port_no] |= BIT(pin);
+		WRITE_BIT(data->general.invert, pin,
```

Comment:
You shouldn't need to handle the invert field. This is now handled in `include/drivers/gpio.h`

Reply:
this was removed in a previous commit